### ISSUE TITLE
seperated scroll

### ISSUE THREAD

**Issue by RokPapez** [spin83/multi-monitors-add-on#2]
Is it possible not to scroll down first and second monitor at once?
For example: on 1st monitor i stay on workspace1 and on second i go to workspace 3.

exit
e

**Comment by spin83** [comment]
It's interesting idea. I look into it.

**Comment by spin83** [comment]
I think that gnome-shell can't display two active workspaces at once.
So separated scroll are not possible.

**Comment by KateAdams** [comment]
It would be awesome if this was actually possible though.  Maybe a bug report/feature request can be submitted upstream (to gnome-shell) to decouple multiple monitor's workspaces?

**Comment by BuZZ-dEE** [comment]
I'm waiting also for this feature.

**Comment by glanc** [comment]
decouple...decouple..decouple!

**Comment by mgallagher** [comment]
Yep, I would love to see this. I'd love to be able to do something like this (from OS X):
http://i.imgur.com/CTGw7yp.gifv

**Comment by geekgonecrazy** [comment]
This would be fantastic.  Has anyone created a feature request on gnome shell?  If so please link it.  Would like to toss my vote in on it.

**Comment by k4kfh** [comment]
This is something I'd love to see as well. GNOME really has a lot of potential in multi-monitor setups if you could choose which monitors had coupled workspaces and which didn't. Even if they were all decoupled, it would be 10x better than what we have now. This is definitely something that gnome-shell needs to address so count me in if anyone submits a feature request that we can all go support.

**Comment by jhasse** [comment]
Here's the upstream bug report: https://bugzilla.gnome.org/show_bug.cgi?id=757399

**Comment by N0NB** [comment]
I added a comment to the upstream BZ as well.  This is a powerful work flow that has been lost to me with recent versions of Xorg or Xfce or both, I'm not sure.

In the mean time, thanks for this extension.

**Comment by gabus** [comment]
Very much needed feature! I don't care if it doesn't follow EWMH specs of implementation is hacky. That would provide better workflow. MacOS is nailing workspaces!

**Comment by N0NB** [comment]
Upstream has move to Gilab and the current issue is: https://gitlab.gnome.org/GNOME/mutter/issues/37#

**Comment by shabtainadav** [comment]
Are there any news about this feature? It's been 2 years since it was opened...
